{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:ibm:qradar_security_information_and_event_manager:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "7.3.3",
            "versionStartIncluding": "7.3.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-4508"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "qradar_security_information_and_event_manager",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "7.3.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "7.3.1"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "7.3.2"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "7.3.3"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "ibm"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM QRadar SIEM 7.3.0 through 7.3.3 uses weak credential storage in some instances which could be decrypted by a local attacker. IBM X-Force ID: 164429."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-522"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "ibm-qradar-cve20194508-info-disc (164429)",
          "refsource": "XF",
          "tags": [
            "VDB Entry",
            "Vendor Advisory"
          ],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/164429"
        },
        {
          "name": "https://www.ibm.com/support/pages/node/1170334",
          "refsource": "CONFIRM",
          "tags": [
            "Patch",
            "Vendor Advisory"
          ],
          "url": "https://www.ibm.com/support/pages/node/1170334"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "acInsufInfo": false,
      "cvssV2": {
        "accessComplexity": "LOW",
        "accessVector": "LOCAL",
        "authentication": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 2.1,
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "vectorString": "AV:L/AC:L/Au:N/C:P/I:N/A:N",
        "version": "2.0"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "LOW",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2020-01-13T19:36Z",
  "publishedDate": "2020-01-10T16:15Z"
}